tests/global_config_x86_64.py

```diff
@@ -28,7 +31,12 @@
 
 fedora_os_matrix = generate_os_matrix_dict(os_name="fedora", supported_operating_systems=["fedora-42"])
 centos_os_matrix = generate_os_matrix_dict(os_name="centos", supported_operating_systems=["centos-stream-9"])
-instance_type_rhel_os_matrix = generate_instance_type_rhel_os_matrix(preferences=["rhel-8", "rhel-9", "rhel-10"])
+instance_type_rhel_os_matrix = generate_instance_type_os_matrix(
+    os_name="rhel", preferences=["rhel.8", "rhel.9", "rhel.10"]
```

Comment:
do we want to save and reuse. called at multiple places.

Reply:
yes, but in a later PR - https://issues.redhat.com/browse/CNV-65113

Reply:
we are using ["rhel.8", "rhel.9", "rhel.10"] twice in this PR so fixing here could be a better choice .
Maybe a have second commit for such cosmetic fixes in the same PR